index.bs

```diff
@@ -1527,6 +1537,479 @@ longer be considered as sanitized.
+    </tr>
+    <tr>
+     <td><code>stroke</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>stroke-dasharray</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>stroke-dashoffset</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>stroke-linecap</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>stroke-linejoin</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>stroke-miterlimit</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>stroke-opacity</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>stroke-width</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>text-anchor</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>text-decoration</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>text-overflow</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>text-rendering</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>transform-origin</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>unicode-bidi</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>vector-effect</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>visibility</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>white-space</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>word-spacing</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+    <tr>
+     <td><code>writing-mode</code></td>
+     <td>Allowed</td>
+     <td></td>
+    </tr>
+
+
+</tbody>
+</table>
+
+### Legend of exclusion criteria for SVG features.
+<ol>
+<li id="svg-attr-1">This attribute is disallowed as a global attribute, we only want to allow it on some elements.
+<li id="svg-attr-2">This attribute is disallowed because we do not allow the SVG `<filter>` element.
+<li id="svg-attr-3">This attribute is disallowed because it is deprecated
+<li id="svg-attr-4">This attribute is disallowed because we do not allow the `<image>` element.
+<li id="svg-attr-5">This attribute is disallowed because we do not allow the `<mask>` element.
+<li id="svg-attr-6">This attribute is disallowed because we do not allow the the element to impact the page layout.
+</ol>
+
+
+## Elements and attributes not allowed in the default config ##  {#default-disallowed-elements}
+
+While the Sanitizer aims to disallow Cross-Site Scripting attacks [[XSS]], the default
+settings are a bit stricter and do not include elements that may change page
+settings, layout or fetch resources from other origins. The following elements
```

Comment:
We don't/can't really differentiate between fetching resources from same vs. other origin, so this seems too specific.